pkg/fetcher/unmarshal_test.go

```diff
@@ -5,52 +5,85 @@ package fetcher //nolint:testpackage // testing internal functions
 import (
 	"reflect"
 	"testing"
+	"time"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
-type testStruct struct {
-	TestString string `fetcherKey:"str"`
-	TestSlice  []int  `fetcherKey:"slice"`
+type testStruct struct { //nolint:govet // changing the order would break the test
```

Comment:
`would break the test` needs more details as to why

Reply:
The tests are using the ordering so better to ignore field alignment complaints otherwise we'll be constantly changing the tests.